app/controllers/api/v1x0/portfolio_items_controller.rb

```diff
@@ -53,6 +53,11 @@ def undestroy
         render :json => item
       end
 
+      def next_name
+        svc = Catalog::NextName.new(params.require(:portfolio_item_id), params[:portfolio_id])
+        render :json => {:next_name => svc.process.next_name }
```

Comment:
Surprised rubocop didn't complain about the lack of a space here: `{:next_name `

Reply:
Oh wow, that is surprising. Fixed!